grunt_tasks/build.js

```diff
@@ -77,17 +77,11 @@ module.exports = function (grunt) {
                     dest: 'clients/web/static/built/jsoneditor'
                 }]
             },
-            bootstrap: {
+            // Provide a copy of ext.min.js under an old name, for compatibility
+            legacy_libs: {
                 files: [{
-                    expand: true,
-                    cwd: 'node_modules/bootstrap/dist/css',
-                    src: ['bootstrap.min.css'],
-                    dest: 'clients/web/static/built'
-                }, {
-                    expand: true,
-                    cwd: 'node_modules/bootstrap-switch/dist/css/bootstrap3',
-                    src: ['bootstrap-switch.min.css'],
-                    dest: 'clients/web/static/built'
+                    src: ['clients/web/static/built/ext.min.js'],
+                    dest: 'clients/web/static/built/libs.min.js'
```

Comment:
I think we should just symlink this rather than copy it. I plan to do this for bootstrap and d3 as well when we move them.